src/file/watcher.go

```diff
@@ -18,6 +18,8 @@ const (
 	Update Op = iota
 	// Remove is a file op where the file is removed
 	Remove
+	// Skip is file op where the remote file matches the local file so is not transferred
+	Skip
```

Comment:
@tanema Does this seem a reasonable approach?

Reply:
Yeah I think it is great and could be used in the watcher as well.